### ISSUE TITLE
image up-to-date check is run every start

### ISSUE THREAD

**Issue by EugenMayer** [EugenMayer/docker-sync#555]
``` 
ok  Checking if a newer unison image is available
```

is done every single start, actually it should do in now and then, not every start. Seems to be introduced with 0.5.2+

**Comment by barat** [comment]
Plus for me it checks it twice?

```
$ docker-sync sync
ok  Checking if a newer unison image is available
ok  No newer image found - current image is up to date.
ok  Checking if a newer unison image is available
ok  No newer image found - current image is up to date.```

**Comment by EugenMayer** [comment]
it checks it per sync endpoint, so you have 2, its twice

**Comment by barat** [comment]
Nope - only one:

docker-sync.yml:
```
version: "2"
options:
  verbose: false
syncs:
  gsdev-sync:
    src: './'
    host_disk_mount_mode: 'cached'
    sync_strategy: 'native_osx'
    sync_args: "-ignore='Path .idea' -ignore='Path .git' -ignore='Name .DS_Store' -ignore='BelowPath cms/app/logs' -ignore='Path cms/app/cache/dev/*' -ignorenot='Path cms/app/cache/dev/translations' -ignorenot='Path cms/app/cache/dev/profiler' -ignorenot='Path cms/app/cache/dev/appDevDebugProjectContainer.xml' -ignorenot='Path cms/app/cache/dev/appDevUrlGenerator.php'"
```
docker-compose.yml
```
(...)
version: "2"
services:
  gs-dev:
    volumes:
      - gsdev-sync:/app:nocopy
      - ~/.ssh:/root/.ssh
(...)
volumes:
  gsdev-sync:
    external: true
```

**Comment by EugenMayer** [comment]
fixed in 0.5.7
